src/containers/Tasks/index.tsx

```diff
@@ -125,6 +160,10 @@ const Tasks: FC = () => {
         />
         <div className="Tasks__left-menu">{renderFilters()}</div>
         <div className="Tasks__task-list">
+          <div className="Tasks__sortby-container">
+            {sortByOption !== SortBy.none ? renderOrder() : null}
```

Comment:
this can use short-circuiting like so:-
sortByOption !== SortBy.none && renderOrder()

Also I think renderOrder can just be replaced by its jsx declaration here like so:-
sortByOption !== SortBy.none && <Icon .... />